docs/conf/portland/2023/writing-day.rst

```diff
@@ -31,10 +32,48 @@ Project Listing
 
 Here are some of the projects that you can work on during Writing Day!
 
-Your Project Here
-~~~~~~~~~~~~~~~~~
+Step-ca
+~~~~~~~
+
+Step-ca is analogous to the popular public web certificate authority, Let’s Encrypt. 
+It is an open-source certificate authority toolkit and ACME server for securely 
+automating certificate issuance and management.
+
+Step-ca is the perfect project to get involved with if you’d like to dive into how 
+TLS and HTTPS work. You can find  `the codebase <https://github.com/smallstep/certificates>`_ and `the docs <https://github.com/smallstep/docs>`_ on GitHub.
+
+We’re looking for volunteers to help polish and make the style more consistent across 
+our most popular doc pages. Our docs are technically correct, but are not very concise. 
+We have opened issues for each of such pages and appriopately labeled them Writing Day.
+
+Review each identified page and consider making the following types of improvements:
+
+- Update and use Semantic Linefeeds consistently
+- Apply guidelines from `Google's Developer Documentation Style Guide <https://developers.google.com/style>`_.
+- Edit for grammar and style issues: convert passive voice to active voice, edit run-on sentences with multiple clauses,
+  reorder concepts lists as needed, etc.
+
+If you come across something you can't fix, you're welcome to create an issue on our repository.
+
+Our developer advocate Linda is at Writing Day! She is available to help you understand exactly what’s
+needed for these tasks and to help work through any problems. We’re so excited to meet you and merge 
+your pull requests!
+
+GitLab Documentation (afternoon-only session)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GitLab is the open-core project behind the platform that empowers people to collaborate 
+on their own projects, primarily to deliver software faster, and more efficiently.
+
+The documentation for GitLab and the GitLab documentation website are open-source 
+and maintained by GitLab team members and our community.
+
+As with previous years, we want to invite participants to contribute! Participants can 
+get a sense of how to contribute to an open-source documentation project, and how to 
+use GitLab. The GitLab platform hosts many open-source projects, so participants will 
+hopefully garner the skills to contribute to other projects! Beginners are welcome 
```

Comment:
```suggestion
hopefully garner the skills to contribute to other projects!

Beginners are welcome 
```

Would appreciate an extra paragraph break here just to make the "Beginners are welcome" part more obvious =)

Otherwise, it looks good to me. Thanks for adding this.